### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by francis3400**
- [ ] I followed basic HTML and CSS syntax
- [ ] Installed eslint, and stylint to fix errors
- [ ] Updated my README.MD file acknowledging Cindy.
- [ ] Used JavaScript interactions for functionalities
- [ ] Made the necessary commits with the right commit statements.
- [ ] Link to video: https://www.loom.com/share/85dd57b6c51e4eb1b71a6fc86774fa6d

### CODE REVIEW

**Review by TadesseDev — COMMENTED** [review]
### Hi @francis3400 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
yPkfDFf

## Invalid Code Review Request 🛑🛑

You have submitted a project without recording a video presenting the features of the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
...
+Read down the page for instructions and requirements
+
+Prerequisites
+Code Editor(VS Studio Code) Web Browser (Google Chrome recommended)
+
+Live Demo
+Live Demo Link
+https://francis3400.github.io/Capstone-Project/
+
+How to Run
```
Comment (uwadonat) [inline README.md]: - [ ] Let's personalize readne file to match what you did in the project. You an follow the given [template.](https://github.com/microverseinc/readme-template/blob/master/README.md)

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @francis3400,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- Descriptive pull request message  ✔️
almoo


## Required Changes ♻️
_Check the comments under the review._

- [ ] Your video should contains a highlight about a piece of the code you find interesting or you are proud of making.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @francis3400 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**